backend/accounting-service/src/jvmMain/kotlin/services/providers/ProviderService.kt

```diff
@@ -281,4 +281,33 @@ class ProviderService(
             throw ex
         }
     }
+
+    suspend fun update(
+        actorAndProject: ActorAndProject,
+        request: ProvidersUpdateSpecificationRequest,
+        ctx: DBContext = db
+    ): ProvidersUpdateSpecificationResponse {
+        val (actor) = actorAndProject
```

Comment:
This should be editable by anyone who has "EDIT" rights to the provider (the resource). You can use the `retrieveBulk` call for this.